GitTfs.VsCommon/TfsHelper.Common.cs

```diff
@@ -618,8 +618,7 @@ public void WithWorkspace(string localDirectory, IGitTfsRemote remote, IEnumerab
             if (!_workspaces.TryGetValue(remote.Id, out workspace))
             {
                 Trace.WriteLine("Setting up a TFS workspace with subtrees at " + localDirectory);
-                var folders = mappings.Select(x => new WorkingFolder(x.Item1, Path.Combine(localDirectory, x.Item2))).ToArray();
-                _workspaces.Add(remote.Id, workspace = Retry.Do(() => GetWorkspace(folders)));
+                _workspaces.Add(remote.Id, workspace = Retry.Do(() => GetWorkspace(mappings.Select(x => new WorkingFolder(x.Item1, Path.Combine(localDirectory, x.Item2))).ToArray())));
```

Comment:
Can you revert this change? It took a second for me to realize there wasn't a change here (or is there?), and it has made the line reallly long. If there is a change, maybe add some wrapping, or make the lambda multiline?